app/vector_icons/sidebar_chat.icon

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
I believe we have a CI check that requires this URL to be HTTPS now. So you might need to update this before landing.

Reply:
Done